airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/legacy_registry.py

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+import json
+
+import pandas as pd
+from dagster import asset, OpExecutionContext
+
+
+from orchestrator.utils.dagster_helpers import OutputDataFrame, output_dataframe
+
+
+from metadata_service.models.generated.ConnectorRegistryV0 import ConnectorRegistryV0
+
+
+GROUP_NAME = "legacy_registry"
```

Comment:
nit: if we're good at naming assets modules we could use `__name__` instead of repeating the module name
```suggestion
GROUP_NAME = __name__
```

Reply:
ooooh I like that!

Reply:
Ah actually this will render as `orchestrator.assets.legacy_registry` which is invalid.

We could parse it but that seems like more effort and bug surface area than its worth.